src/Nuget.PackageIndex/NupkgLocalPackageLoader.cs

```diff
@@ -40,10 +54,23 @@ internal class NupkgLocalPackageLoader : ILocalPackageLoader
         }
 
         /// <summary>
-        /// Returns metadata info for given nupkg file
+        /// Returns metadata info for given nupkg file. The logic for picking right
+        /// reference asemblies is:
+        /// 
+        /// Action Path                                     Target
+        /// ====== ======================================== ======
+        /// +      lib\assembly.dll                         any
+        /// +      lib\fx\asembly.dll                       fx
+        /// -      lib\tools\assembly.dll                   -
```

Comment:
Since when can a `lib` folder have a `tools` or `content` subfolder? Is that a typo here?